docs/admin/troubleshooting.md

```diff
@@ -919,6 +919,45 @@ pool header correct
 0
 ```
 
+## Syslog
+
+RAS events are printed to the Syslog by 'daos_server' processes via the golang standard library API.
```

Comment:
nit: The name of the language is "Go", not "golang". We should try to be consistent.